server/chat.ts

```diff
@@ -1720,7 +1720,7 @@ export const Chat = new class {
 			if (typeof entry !== 'function') continue;
 
 			const handlerCode = entry.toString();
-			entry.requiresRoom = /\bthis\.requires?Room\(/.test(handlerCode);
+			entry.requiresRoom = /requireRoom\((?:'|"|`)(.*?)(?:'|"|`).\)/.exec(handlerCode)?.[1] || false;
```

Comment:
Will this be `true` for `requireRoom()` with no arguments?

Reply:
Looks like no. That'll need to be fixed.

Reply:
Fixed.